src/components/common/notification-center.svelte

```diff
@@ -46,6 +58,39 @@
       <div class="title">No notifications</div>
     </div>
   {:else}
-    <p>List notifications here</p>
+    <SimplebarList classname="notification-list">
+      {#each notifications as notification (notification.id)}
+        <li class="notification">
+          <span class="content">
+            {#each getNotificationContent(notification) as fragment}
+              {#if typeof fragment === 'string'}
+                {fragment}
+              {:else if 'bold' in fragment}
+                <strong>{fragment.text}</strong>
+              {:else}
+                <a href="{fragment.url}" rel="prefetch">
+                  {fragment.text}
+                </a>
+              {/if}
+            {/each}
+          </span>
+          <Button
+            isRound
+            tooltip="Mark as read"
+            classname="mt"
+            on:click={() => readNotification(notification.id)}
+          >
+            <svg src="images/icons/check.svg" class="icon" />
+          </Button>
+          <time datetime={notification.timestamp}>
```

Comment:
This would look better than the ISO string:
```suggestion
          <time datetime={new Date(notification.timestamp).toLocaleString()}>
```

Reply:
No, this attribute is supposed to contain an ISO string. It's for programmatic parsing
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Reply:
My bad. I thought it was being used as a tooltip.